.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: Close stale PRs
+
+on:
+  schedule:
+  - cron: "0 * * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1.1.0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: |
+          We're closing this PR because it hasn't been updated in a while.
+          This isn't a judgement on the merit of the PR in any way. It's just
+          a way of keeping the PR queue manageable.
+
+          If you'd like to revive this PR, please reopen it!
+        days-before-stale: 3650
```

Comment:
This would be 10 years? 6 months is OK, or maybe less. 1000 days? I also don't think it needs to warn.

Reply:
> I also don't think it needs to warn.

Are you talking about `days-before-close`?

Reply:
Both. I think this number has to be lower. Oops, I meant 100 days not 1000 BTW. Also I don't think we have to warn before closing.